### ISSUE TITLE
SETTING DEADLINE TIME

### ISSUE THREAD

**Issue by knana1662** [vip20/vscode-agile-board#12]
I tried to use the latest update but tried to set a deadline for the board and made the time 1 minute ahead of my current time to see that the time is up, which worked well but remained silent and also, I had to close it and open it to see that the deadline time is up. Now the questions are as follows:

1.    when the deadline time is created. does it or is it supposed to produce a sound or play a sound to show that the time is up, or it remains silent.
 
2.  If it remains silent then I would like to ask for a pop-up notification to display the details (Board Title and Deadline Time).

3.   And also, can a sound be added to it, to show that the deadline time is up. Thank You.

NB:
Make number 2 the main priority and number 3 an option.

**Comment by vip20** [comment]
thank you for trying it. Please rate the plugin in store.

To answer your questions
1. No it doesnt show any notifications or play sound. 
2. Popup notification based on time might consume background resources. Also only works when VS Code is open.
3. FYI point 2.

Hope these answer your questions.

**Comment by vip20** [comment]
Checking every second is the time is satisfies is like compromising your machine's memory. Imagine this happening on a lower end PC

**Comment by knana1662** [comment]
Yes, I know that, but can the popup not be implemented to the vscode's notification?

**Comment by vip20** [comment]
It's the same right, all the tasks has to be scanned and event has to be verified each second and if the condition satisfies a notification has to be triggered. That will take up more resources. I am looking at OS scheduler to trigger a system notification. This has a learning curve and is different for each platform.

**Comment by knana1662** [comment]
Ok thanks for the update also✌👍

**Comment by vip20** [comment]
I tried a lot and didnt succeed. I'll update if I come up with anything
